lib/message_builder.rb

```diff
@@ -94,11 +98,84 @@ def no_pull_requests
     render "no_pull_requests"
   end
 
+  def prs_count_message
+    if @all_prs_count > 1
+      "You have #{@all_prs_count} Dependabot PRs open on the following apps:"
+    else
+      "You have 1 Dependabot PR open on the following app:"
+    end
+  end
+
+  def repo_details(repo)
+    pr_count = repo[:pr_count]
+
+    if pr_count > 1
+      "oldest: #{days_plural(repo[:oldest_pr])}, newest: #{days_plural(repo[:newest_pr])})"
+    else
+      "opened #{days_plural(repo[:oldest_pr])})"
+    end
+  end
+
+  def security_alerts_intro
+    if @all_alerts_count.nil?
+      ":warning: Failed to get security alerts for your repos. :warning:\n" \
```

Comment:
We're missing accompanying tests for [message_builder_spec.rb](https://github.com/alphagov/seal/blob/main/spec/message_builder_spec.rb). You could probably just get away with one or two new tests.